common/src/main/java/io/netty/util/internal/InternalThreadLocalMap.java

```diff
@@ -164,13 +164,17 @@ public int size() {
     }
 
     public StringBuilder stringBuilder() {
-        StringBuilder builder = stringBuilder;
-        if (builder == null || builder.capacity() > STRING_BUILDER_MAX_CAPACITY) {
```

Comment:
@normanmaurer That doesn't seem necessary to me but if you think it is I would be happy to add it.